File: backends/php-laravel/app/Domain/Staff/ValueObjects/StaffVo.php
<?php
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
namespace App\Domain\Staff\ValueObjects;

use App\Support\Traits\Getter;
use App\Support\ValueObjects\AbstractValueObject;

/**
 * スタッフValueObjectクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Domain\Staff\ValueObjects
 *
 * @method int|null getId()
 * @method string|null getName()
 * @method string|null getEmail()
 * @method string|null getAvatar()
 * @method int|null getRole()
 * @method int|null getStatus()
 */
class StaffVo extends AbstractValueObject
{
    use Getter;

    private ?int $id = null;
    private ?string $name = null;
    private ?string $email = null;
    private ?string $avatar = null;
    private ?int $role = null;
    private ?int $status = null;
}

File: backends/java-spring/src/main/java/com/authorization/domain/staff/valueobjects/StaffVo.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.staff.valueobjects;

import com.authorization.support.valueobjects.AbstractValueObject;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * スタッフValueObjectクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class StaffVo extends AbstractValueObject {

    private Long id;
    private String name;
    private String email;
    private String avatar;
    private Integer role;
    private Integer status;
}
